app_flutter/lib/service/appengine_cocoon.dart

```diff
@@ -420,6 +420,9 @@ class AppEngineCocoonService implements CocoonService {
       ..requiredCapabilities.add(objectRequiredCapabilities.toString())
       ..reservedForAgentId = taskData['ReservedForAgentID']
       ..stageName = taskData['StageName']
-      ..status = taskData['Status'];
+      ..status = taskData['Status']
+      ..buildNumberList = taskData['BuildNumberList'] ?? ''
+      ..builderName = taskData['BuilderName'] ?? ''
+      ..luciPoolName = taskData['LuciPoolName'] ?? '';
```

Comment:
This should be bucket instead of pool name.

Reply:
Done.